File: Code/Components/DanpheEMR.ServerModel/InventoryModels/InventoryReportModel/DetailStockLedgerModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel.InventoryModels.InventoryReportModel
{
    public class DetailStockLedgerModel
    {
        public DateTime? TransactionDate { get; set; }
        public double? ReceiptQty { get; set; }
        public decimal? ReceiptRate { get; set; }
        public double? ReceiptAmount { get; set; }
        public double? IssueQty { get; set; }
        public decimal? IssueRate { get; set; }
        public double? IssueAmount { get; set; }
        public double BalanceQty { get; set; }
        public decimal BalanceRate { get; set; }
        public decimal BalanceAmount { get; set; }
        public int ReferenceNo { get; set; }
        public string Store { get; set; }
        public string Username { get; set; }
        public string Remarks { get; set; }

    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/inventory/shared/detail-stock-ledger.model.ts

import {
    NgForm,
    FormGroup,
    FormControl,
    Validators,
    FormBuilder,
    ReactiveFormsModule
} from '@angular/forms'
import * as moment from 'moment';

export class DetailStockLedger {
    public ItemId: number = null;
    public TransactionDate: string = "";
    public ReceiptQty: number = 0;
    public ReceiptRate: Number = 0;
    public ReceiptAmount: number = 0;
    public IssueQty: number = 0;
    public IssueRate: number = 0;
    public IssueAmount: number = 0;
    public BalanceQty: number = 0;
    public BalanceRate: number = 0;
    public BalanceAmount: number = 0;
    public ReferenceNo: number = 0;
    public Store: string = "";
    public Username: string = "";
    public Remarks: string = "";
    public FromDate: string = moment().format('YYYY-MM-DD');;
    public ToDate: string = moment().format('YYYY-MM-DD');;





}